e2e/whisper/mailservice_test.go

```diff
@@ -0,0 +1,57 @@
+package whisper
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/status-im/status-go/e2e"
+	"github.com/status-im/status-go/geth/node"
+	"github.com/status-im/status-go/geth/params"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestMailServiceSuite(t *testing.T) {
+	suite.Run(t, new(MailServiceSuite))
+}
+
+type MailServiceSuite struct {
+	e2e.NodeManagerTestSuite
+}
+
+func (s *MailServiceSuite) SetupTest() {
+	s.NodeManager = node.NewNodeManager()
+}
+
+func (s *MailServiceSuite) TestShhRequestMessagesRPCMethodAvailability() {
```

Comment:
Because it checks if `shh_requestMessages` is available through `HTTP` and inproc interfaces. Even though both errors are the same I don't think we should test for their equality as they are not connected at all.